packages/react-core/src/next/components/DualListSelector/DualListSelectorPane.tsx

```diff
@@ -0,0 +1,245 @@
+import { SearchInput } from '../../../components/SearchInput';
+import cssMenuMinHeight from '@patternfly/react-tokens/dist/esm/c_dual_list_selector__menu_MinHeight';
+
+/** Acts as the container for a list of options that are either available or chosen,
+ * depending on the pane type (available or chosen). A search input and other actions,
+ * such as sorting, can also be passed into this sub-component.
+ */
+
+export interface DualListSelectorPaneProps extends Omit<React.HTMLProps<HTMLDivElement>, 'title'> {
+  /** Additional classes applied to the dual list selector pane. */
+  className?: string;
+  /** A dual list selector list or dual list selector tree to be rendered in the pane. */
+  children?: React.ReactNode;
+  /** Flag indicating if this pane is the chosen pane. */
+  isChosen?: boolean;
+  /** Status to display above the pane. */
+  status?: string;
+  /** Title of the pane. */
+  title?: React.ReactNode;
+  /** A search input placed above the list at the top of the pane, before actions. */
+  searchInput?: React.ReactNode;
+  /** Actions to place above the pane. */
+  actions?: React.ReactNode[];
+  /** Id of the pane. */
+  id?: string;
+  /** @hide Options to list in the pane. */
+  options?: React.ReactNode[];
+  /** @hide Options currently selected in the pane. */
+  selectedOptions?: string[] | number[];
+  /** @hide Callback for when an option is selected. Optionally used only when options prop is provided. */
+  onOptionSelect?: (
+    event: React.MouseEvent | React.ChangeEvent | React.KeyboardEvent,
+    index: number,
+    isChosen: boolean,
+    id?: string,
+    itemData?: any,
+    parentData?: any
+  ) => void;
+  /** @hide Callback for when a tree option is checked. Optionally used only when options prop is provided. */
+  onOptionCheck?: (
+    event: React.MouseEvent | React.ChangeEvent<HTMLInputElement> | React.KeyboardEvent,
+    isChecked: boolean,
+    itemData: DualListSelectorTreeItemData
+  ) => void;
+  /** @hide Flag indicating a dynamically built search bar should be included above the pane. */
+  isSearchable?: boolean;
+  /** Flag indicating whether the component is disabled. */
+  isDisabled?: boolean;
+  /** Callback for search input. To be used when isSearchable is true. */
+  onSearch?: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  /** @hide A callback for when the search input value for changes.  To be used when isSearchable is true. */
+  onSearchInputChanged?: (event: React.FormEvent<HTMLInputElement>, value: string) => void;
+  /** @hide Callback for search input clear button */
+  onSearchInputClear?: (event: React.SyntheticEvent<HTMLButtonElement>) => void;
+  /** @hide Filter function for custom filtering based on search string. To be used when isSearchable is true. */
+  filterOption?: (option: React.ReactNode, input: string) => boolean;
+  /** @hide Accessible label for the search input. To be used when isSearchable is true. */
```

Comment:
Are all of these hidden props still needed? is the search still built dynamically? In the basic search example it seems you're still building it yourself?

Reply:
thanks for the comments and explanations @nicolethoen. i've removed or updated the code/docs to more accurately reflect the composable model.